modules/nf-core/wipertools/fastqgather/tests/main.nf.test

```diff
@@ -0,0 +1,66 @@
+nextflow_process {
+
+    name "Test Process WIPERTOOLS_FASTQGATHER"
+    script "../main.nf"
+    process "WIPERTOOLS_FASTQGATHER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "wipertools"
+    tag "wipertools/fastqgather"
+    tag "untar"
+
+    test("merge two gzipped fastq - fastq.gz") {
+
+        config "./base.config"
+
+        when {
+            params {
+                outdir   = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [
+                    [id: 'fastq_chunks'],
+                    [file('https://raw.githubusercontent.com/nf-core/test-datasets/fastqrepair/testdata/pp_1-of-2_ss.fastq.gz', checkIfExists: true),
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/fastqrepair/testdata/pp_2-of-2_ss.fastq.gz', checkIfExists: true)]
```

Comment:
can you correct the paths please? :)

Reply:
OK, I've corrected the paths in this file. Later tomorrow, I'll edit the other files also. Thanks for catching this point.